src/app.js

```diff
@@ -0,0 +1,51 @@
+// import isYes function to use
+import { isYes } from './is-yes.js';
+
+//globally available variables
+const myButton = document.getElementById('action-button');
+let correctAnswers = 0;
+const hiddenMessage = document.querySelector('.hidden');
+
+
+//button-click handler function, inputs string user name, answers 3 y/n Qs, returns string about score
+myButton.addEventListener = ('click', () => {
```

Comment:
You were so very close with this refactor. To have full functionality, though, you will need to have code that looks like: ``` myButton.addEventListener('click', () => {  ```

So, you just need to get rid of the "=".

Reply:
I fixed this issue